test/pensions/config/marriageHistory.unit.spec.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester, submitForm } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/pensions/config/form';
+
+function fillData(formDOM, id, value) {
+  ReactTestUtils.Simulate.change(formDOM.querySelector(id), {
+    target: {
+      value
+    }
+  });
+}
+
+describe('Pensions marriage history', () => {
+  const marriageHistory = formConfig.chapters.householdInformation.pages.marriageHistory;
+  const uiSchema = marriageHistory.uiSchema.marriages.items;
+  const schema = marriageHistory.schema.properties.marriages.items;
+
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(19);
+  });
+
+  describe('hideIf current marriage', () => {
+    const hideIfCurrentMarriage = marriageHistory.uiSchema.marriages.items['view:pastMarriage']['ui:options'].hideIf;
+
+    it('hides if married and last', () => {
+      const result = hideIfCurrentMarriage({
+        maritalStatus: 'Married',
+        marriages: [{}]
+      }, 0);
+
+      expect(result).to.be.true;
+    });
+
+    it('does not hide if married and not last', () => {
+      const result = hideIfCurrentMarriage({
+        maritalStatus: 'Married',
+        marriages: [{}, {}]
+      }, 0);
+
+      expect(result).to.be.false;
+    });
+
+    it('does not hide if not married', () => {
+      const result = hideIfCurrentMarriage({
+        marriages: [{}]
+      }, 0);
+
+      expect(result).to.be.false;
+    });
+  });
+
+  describe('page title', () => {
+    const pageTitle = marriageHistory.title;
+    it('uses word for index', () => {
+      expect(pageTitle({}, { pagePerItemIndex: 0 })).to.equal('First marriage');
+    });
+    it('uses number when over ten', () => {
```

Comment:
Perhaps this could be rephrased, "when contains over ten items/length is greater than or equal to ten" or "when at index ten or greater"